vendor/github.com/opencontainers/selinux/go-selinux/BUILD.bazel

```diff
@@ -0,0 +1,9 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["selinux_stub.go"],
+    importmap = "kubevirt.io/kubevirt/vendor/github.com/opencontainers/selinux/go-selinux",
+    importpath = "github.com/opencontainers/selinux/go-selinux",
+    visibility = ["//visibility:public"],
+)
```

Comment:
@rmohr tests are failing because of this file that should look like the below, but it keeps changing after each make generate... 
What did I do wrong? :) 

load("@io_bazel_rules_go//go:def.bzl", "go_library")

go_library(
    name = "go_default_library",
    srcs = [
        "selinux_linux.go",
        "xattrs.go",
    ],
    importmap = "kubevirt.io/kubevirt/vendor/github.com/opencontainers/selinux/go-selinux",
    importpath = "github.com/opencontainers/selinux/go-selinux",
    visibility = ["//visibility:public"],
    deps = select({
        "@io_bazel_rules_go//go/platform:linux": [
            "//vendor/golang.org/x/sys/unix:go_default_library",
        ],
        "//conditions:default": [],
    }),
)

Reply:
What was the issue?

Reply:
As I understand it and my understanding is very limited :) ,
 `# gazelle:build_tags selinux` was missing from the BUILD.bazel 
and the build command should have looked like `build --define gotags=selinux`